### ISSUE TITLE
"ammonia"/fluid synthesizer

### ISSUE THREAD

**Issue by Wisebiscuit** [Silfryi/PracticalEngineering-AdvancedTechnology#1]
I can't get the fluid synthesizer to work with the custom recipes. Producing ammonia (useless at my early stage of the game) works but the custom recipes for plastic/sulfur trioxide do not work. Also verified I could not get them working through twitch launcher version (normally use MultiMC) on creative mode. Other minor issues I don't want to open a separate ticket for: NEI display for  fluid labels on synthesizer is messed up (offset), and probably electrolyzer as well (can't see fluid names at all). When installing through either twitch launcher or MultiMC, launching the game prompts that forge-multipart is missing; had to go acquire it myself.

Just trying to help out, thanks.

**Comment by voidsong-dragonfly** [comment]
Sadly, all of these issues other than the FMP one which I am not sure _why_ it occurs are on Reika's end. He knows about the Fluid Synth problems but hasn't found a way to fix them yet, and that NEI integration is on his end not mine.
Going to leave this open until Reika finds a way around the Fluid Synth recipe issues, though.
[Processing.txt](https://github.com/Silfryi/PracticalEngineering-AdvancedTechnology/files/4996208/Processing.txt)
Try replacing the fluid synthesizer recipes with this, it'll _work_ but it consumes cobblestone to make the recipes function (I have no clue why these but not others work), and some other consumption rates also don't work correctly.

**Comment by Wisebiscuit** [comment]
Ok, thanks for the help. I'll let you know if I find any more issues. I usually don't attempt to report issues and instead abandon broken modpacks, but there aren't enough good rotarycraft modpacks and you seem to be active and care about the pack.

**Comment by voidsong-dragonfly** [comment]
Yeah, a lot of the stuff that I'd like to fix is waiting on Reika in that department, I've gotten a lot of the other bugs ironed out.
